linkis-computation-governance/linkis-entrance/src/main/scala/org/apache/linkis/entrance/scheduler/EntranceParallelConsumerManager.scala

```diff
@@ -86,9 +86,9 @@ class EntranceParallelConsumerManager(maxParallelismUsers: Int, schedulerName: S
               .info(
                 "group {} refresh maxAllowRunningJobs => {}/{}={}",
                 group.getGroupName,
-                group.getMaxRunningJobs,
-                validInsCount,
-                maxAllowRunningJobs
+                group.getMaxRunningJobs.toString,
+                validInsCount.toString,
+                maxAllowRunningJobs.toString
```

Comment:
use array() to wrap the four params

Reply:
done

Reply:
array() will be as a parameter
![image](https://user-images.githubusercontent.com/29919212/223662000-f9e56796-308d-4eb0-90e2-7bf63ec02382.png)